src/main/java/Event.java

```diff
@@ -0,0 +1,75 @@
+import org.json.simple.JSONObject;
+
```

Comment:
I also only just realised that there isn't javadocs in most classes/methods. (Which demonstrates how good your naming is)